src/test/scala/io/iohk/ethereum/blockchain/sync/SyncControllerSpec.scala

```diff
@@ -332,6 +342,123 @@ class SyncControllerSpec extends FlatSpec with Matchers {
     peer4.expectMsg(PeerActor.Unsubscribe)
   }
 
+  it should "broadcast all blocks if they were all valid" in new TestSetup {
```

Comment:
@AlanVerbner Looks like the test is in the right place. We should be able to stub ledger execution.